src/components/__tests__/HierarchicalMenu.js

```diff
@@ -744,10 +744,12 @@ describe('custom showMoreLabel render', () => {
     expect(wrapper.html()).toMatchSnapshot();
   });
 
-  it('renders correctly with a custom show more label toggled', () => {
+  it('renders correctly with a custom show more label toggled', async () => {
     __setState({
       ...defaultState,
-      toggleShowMore: () => wrapper.setData({ state: { isShowingMore: true } }),
+      toggleShowMore: async () => {
+        await wrapper.setData({ state: { isShowingMore: true } });
```

Comment:
this needs to be awaited?

Reply:
https://github.com/algolia/vue-instantsearch/pull/1006/commits/80c71c0bead6aff9befe69c37ec0ffbb44159a77
absolutely not